versions/3.0.0.md

```diff
@@ -1821,7 +1821,7 @@ The key value used to identify the callback object is an expression, evaluated a
 ##### Patterned Fields
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="callbackExpression"></a>{expression} | [Path Item Object](#path-item-object) | A Path Item Object used to define a callback request and expected responses.  A [complete example](../examples/v3.0/callback-example.yaml) is available.
+<a name="callbackExpression"></a>{expression} | [Path Item Object](#path-item-object) | A Path Item Object used to define a callback request and expected responses.  A [complete example](https://github.com/OAI/learn.openapis.org/tree/main/examples/v3.0/callback-example.yaml) is available.
```

Comment:
I'd prefer to link to wherever we host these examples on the actual learn.openapis.org site.  One thing we're trying to do by moving these examples out of this repo is to stop linking into GitHub repos and only treat spec.openapis.org and learn.openapis.org as official published locations.  This might require getting them published on the learn site first if they're not on a page and are instead just sitting there right now.

Reply:
@handrews For example, something like this link https://learn.openapis.org/examples/v3.1/tictactoe.yaml? If yes, then it seems that we are lucky since the whole folder is already deployed.
Although I have one question, I understand why we want to point to the website but are we sure that this is currently the best thing we should be doing? I am asking because Github formats the file better than the browser itself.

Reply:
Hm... we should add something to the Learn site's build to get them to render as syntax highlighted HTML.  I'd rather take the extra time to do that than to move to an interim place, have people get used to that, and then have to convince them to _stop_ looking at the interim place.  Particularly because we obviously need to have these in GitHub _somewhere_.  I just think folks should look at the Learn and Spec sites and ideally stop looking at GitHub unless they need to do work in the repository (including filing issues, commenting, etc.).

But I am totally making this plan up as I type it, and others might feel differently.

Reply:
With a bit of experimentation, it looks like we could move the JSON and YAML files under _includes and do something like this for each one:

```
---
layout: default
title: Tic-Tac-Toe Example (YAML)
parent: Examples
nav_order: 1
has_toc: false 
---

{% highlight yaml %}
{% include examples/v3.1/tictactoe.yaml %}
{% endhighlight %}
```